test/functional/tools/gzipped_inputs.xml

```diff
@@ -10,8 +10,8 @@ cat '$input1' > '$out_file1'
     </outputs>
     <tests>
         <test>
-            <param name="input1" value="4.bed.gz" />
-            <output name="out_file1" file="4.bed" />
+            <param name="input1" value="4.bed.gz" ftype="bed"/>
```

Comment:
I think the test is more "interesting" without the `ftype`, i.e. with Galaxy automagically inferring the input file type.

Reply:
Indeed

Reply:
```suggestion
            <param name="input1" value="4.bed.gz" />
```